wwwapp/models.py

```diff
@@ -59,6 +59,47 @@ class WorkshopUserProfile(models.Model):
     def __str__(self):
         return '%s: %s, %s' % (self.year, self.user_profile, self.status)
 
+class PESELField(models.CharField):
+    """PESEL field, with checksum verification."""
+
+    def validate(self, pesel: str, user_info: 'UserInfo') -> None:
+        if not pesel:
+            raise ValidationError('PESEL type is {})'.format(type(pesel)))
+
+        # https://en.wikipedia.org/wiki/PESEL#Format
+        if len(pesel) != 11:
```

Comment:
This makes the PESEL field required (length of 0 is bad) - may cause problems for users when trying to edit profile before all data is filled in.